addons/website/static/src/components/dialog/seo.js

```diff
@@ -369,13 +590,165 @@ class TitleDescription extends Component {
     }
 }
 
+export class SeoChecks extends Component {
+    static template = "website.SeoChecks";
+    static components = {
+        CheckBox,
+    };
+    static props = {};
+
+    async setup() {
+        this.website = useService("website");
+        this.seoContext = useState(seoContext);
+        const {
+            metadata: { mainObject, seoObject },
+        } = this.website.currentWebsite;
+        this.object = seoObject || mainObject;
+        this.state = useState({
+            altAttributes: [],
+            checkingLinks: false,
+            checkingLink: false,
+            checkedLinks: false,
+            counterLinks: 0,
+            totalLinks: 0,
+            headingsScan: [],
+        });
+        this.imgUpdated = this.imgUpdated.bind(this);
+        onMounted(async () => {
```

Comment:
`onWillStart` instead of `onMounted` here no ? :thinking:

Reply:
Definitely right! :) thanks.